### PR TITLE
SQSCANNER-68 - Automatically trigger docker release on new CLI release

### PR DESCRIPTION
**Pull request by mark-rekveld-sonarsource**
New release action

### CODE REVIEW

**Comment by mark-rekveld-sonarsource** [comment]
As discussed this action should only be missing the release trigger to the sonar-scanner-cli-docker repo

**Comment by mark-rekveld-sonarsource** [comment]
@malena-ebert-sonarsource I added the trigger to the docker repo in df41e30

**Review by malena-ebert-sonarsource — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,32 +1,71 @@
...
+      uses: Ilshidur/action-slack@2.0.0
+      env:
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+      with:
+        args: "Release successful for {{ GITHUB_REPOSITORY }} by {{ GITHUB_ACTOR }}"
+    - name: Create Release for Docker Image
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
[the remark this answers is not in the document]
Comment (mark-rekveld-sonarsource) [reply .github/workflows/release.yml]: Addressed in 912e42f

**Review by malena-ebert-sonarsource — APPROVED** [review]
LGFM

**Pull request merged**